release-notes/README.md

```diff
@@ -14,12 +14,16 @@ You can find release notes for all releases, including out-of-support releases,
 [8.0.7]: 8.0/8.0.7/8.0.7.md
 [6.0.32]: 6.0/6.0.32/6.0.32.md
 
-## Release Information
+## Download .NET
 
 * [Download .NET](https://dotnet.microsoft.com/download/dotnet)
-* [Releases Index][releases-index.json] -- Index for all release channels in JSON format
-* [dotnet-install scripts](https://learn.microsoft.com/dotnet/core/tools/dotnet-install-script)
```

Comment:
What's the thinking behind removing the install script link? I think I understand that it's the only item directly related to installation so is a bit out of place.

Reply:
I just moved it up.